CompilerSource/ipc/Room.fbs

```diff
@@ -0,0 +1,113 @@
+
+table RoomInstance {
+  id:int;
+
+  x:int;
+  y:int;
+  object_id:int;
+
+  flip_x:bool;
+  flip_y:bool;
+  scale_x:double;
+  scale_y:double;
+  rotation:double;
+  color:int;

```

Comment:
In the UI, the user selects both a color and an alpha for _image_blend_ and _image_alpha_ respectively. But for our buffers we can combine them into a single int, which GMX does. Is this right or should we make alpha a separate field?